pyfr/plugins/fluidforce.py

```diff
@@ -136,3 +183,32 @@ def __call__(self, intg):
 
             # Flush to disk
             self.outf.flush()
+
+    def stress_tensor(self, u, du):
+        c = self._constants
+
+        # Density and gradient of density
+        rho = u[0]
+        rcprho = 1/u[0]
```

Comment:
NumPy is slow enough that you can feel free to just divide as and when you need to.